docs/api-guide/routers.md

```diff
@@ -167,12 +167,23 @@ This behavior can be modified by setting the `trailing_slash` argument to `False
 
 Trailing slashes are conventional in Django, but are not used by default in some other frameworks such as Rails.  Which style you choose to use is largely a matter of preference, although some javascript frameworks may expect a particular routing style.
 
-The router will match lookup values containing any characters except slashes and period characters.  For a more restrictive (or lenient) lookup pattern, set the `lookup_value_regex` attribute on the viewset.  For example, you can limit the lookup to valid UUIDs:
+By default the URLs created by `SimpleRouter` use regular expressions. This behavior can be modified by setting the `use_regex_path` argument to `False` when instantiating the router, in this case [path converters][path-converters-topic-reference] are used. For example:
+
+    router = SimpleRouter(use_regex_path=False)
+
+**Note**: `use_regex_path=False` only works with Django 2.x or above, since this feature was introduced in 2.0.0. See [release note][simplified-routing-release-note]
+
+
+The router will match lookup values containing any characters except slashes and period characters.  For a more restrictive (or lenient) lookup pattern, set the `lookup_value_regex` attribute on the viewset or `lookup_value_converter` if using path converters.  For example, you can limit the lookup to valid UUIDs:
 
     class MyModelViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
         lookup_field = 'my_model_id'
         lookup_value_regex = '[0-9a-f]{32}'
 
+    class MyPathModelViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
+        lookup_field = 'my_model_uuid'
+        lookup_value_converter = 'uuid'
```

Comment:
can we use better name for lookup_value_converter?any better suited name instead of converter?

Reply:
I used "converter" since django calls these elements [path converters](https://docs.djangoproject.com/en/stable/topics/http/urls/#path-converters), maybe we can use the full expanded name to have `lookup_value_pathconverter` (or `lookup_value_path_converter`).

Any suggestion is welcome.

Reply:
I would inclined to @tomchristie for a better insight here as I'm bit confused about naming

Reply:
what about just lookup_value_path? this alighned with the name lookup_value_regex

Reply:
I think that it would be better to use the same naming of django.

In this case it was named "converter" thus an attribute which should define one should be named after that. In this way it is easier to find references to what this element does.

Reply:
Agree on using `path converter` or `converter`